main.yml

```diff
@@ -19,7 +20,7 @@ advisor:
   source_repo: https://github.com/RedHatInsights/insights-advisor-frontend
 
 ansible:
-  title: Red Hat Ansible Automation Platform
+  title: Ansible Automation Platform
```

Comment:
There are lots of additional changes in this PR. For instance, multiple changes to the titles of apps. Were these intentional?

Reply:
Yup, they are intentional we don't want to show Red Hat anymore, if you want I can open new PR just for these changes.

Reply:
No it's OK! Just wanted to make sure it was intentional.